lib/cogy.rb

```diff
@@ -165,6 +166,6 @@ def self.configure
   # @example
   #   Cogy.helper(:foo) { |env| env["COGY_POINT"].reverse }
   def self.helper(name, &blk)
-    define_singleton_method(name, blk)
+    Context.class_eval { define_method(name, blk) }
```

Comment:
This seems a bit ugly (defining a class method in another module), but we can rethink that in the helper's revamp.